dev/Playground.vue

```diff
@@ -1,15 +1,28 @@
 <template>
-  <div>
-  </div>
+    <div style="width: 50%">
+      <b-select
+        :selectOptions="[{id: 1, displayName: 'This'}, {id: 2, displayName: 'Is'}, {id: 3, displayName: 'An Example'}]"
+        id="example-id-3"
+        v-model="selected"
+        selectLabel="Select Example"
+        required
+      />
+    </div>
+
 </template>
         
 <script>
+  import BSelect from '../src/components/BSelect/BSelect.vue'
+
+  const selected = 3
 
   export default {
     components: {
+      BSelect
```

Comment:
You shouldn't need to do this if you export it from `src/components/index.js`.  This is preferred so all Buoy components are immediately available.

`export { default as BSelect } from './BSelect'`

Reply:
oh tahts not needed, its just an artifact of messing with stuff, I'll fix

Reply:
Cool was just making sure it was exported in the Buoy library.  Some previous merges have left some components out.  Can't really see that file in this PR.